ridemyway/tests/tests_v2/test_auth/test_signup.py

```diff
@@ -23,7 +23,6 @@ class TestSignUp(V2BaseTest):
     """
 
     def test_driver_can_signup_successfully(self):
-        print(VALID_DRIVER)
         self.response = self.client().post(SIGNUP, data=VALID_DRIVER)
         self.assertEqual(self.response.status_code, 201,
```

Comment:
Add edge cases to your tests e.g if name filed is empty

Reply:
I've actually covered that case for signup. Thanks for the review as it has helped me notice that case is not covered in the login endpoint.

Reply:
I'll make the suggested changes in the login PR #30